First_section.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="First_section.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <section>
+        <header class="toolbar">
+          <h1 id="Logo"><a href="." rel="home">My Logo</a></h1>
+          <nav><img id="menu" src="union.png" /></nav>
+        </header>
+      </section>
+
+      <main class="content">
+        <span class="text_header">
+          <p id="welcome">I am Christian Glad to see you!</p>
+        </span>
+
+        <p id="description">
+          I am software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+
+        <span class="social">
+          <h2>LET'S CONNECT</h2>
+        </span>
+        <section class="icons">
+          <img id="twitter" src="Vector.png" />
+          <img id="linkedin" src="Linkedin icon.png" />
+          <img id="vector1" src="Vector (1).png" />
+          <img id="vector2" src="Vector (3).png" />
+          <img id="vector3" src="Vector (2).png" />
+        </section>
```

Comment:
Required:

- [ ] As mentioned by the previous reviewer, kindly replace the `section` tag with a `ul` tag and wrap each image in an `li` with `a` inside it. This way you will have an unordered list with list items, each item is a link 👍